tests/10-issue.js

```diff
@@ -26,95 +26,284 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
+            'MUST express a type property that includes the ' +
+            'BitstringStatusListCredential value.',
+          async function() {
+            statusListCredential.hasOwnProperty('type',
+              'Expected credential status to have a type.');
+            statusListCredential.type.should.include.members([
+              'BitstringStatusListCredential'
+            ],
+            'Expected credential status type to include ' +
+            'BitstringStatusListCredential.');
+          }
+          );
+          it('The type of the credential subject, which is the status list, ' +
+            'MUST be BitstringStatusList.',
+          async function() {
+            statusListCredential.credentialSubject.hasOwnProperty('type',
+              'Expected credentialSubject to have a type.');
+            const credentialSubjectType = typeof (
+              statusListCredential.credentialSubject.type);
+            credentialSubjectType.should.be.oneOf(['string', 'array'],
+              'Expected credential status type to be a string or an array.');
+            if(credentialSubjectType === 'string') {
+              statusListCredential.credentialSubject.type = [
+                statusListCredential.credentialSubject.type];
+            }
+            statusListCredential.credentialSubject.type.should.include.members([
+              'BitstringStatusList'
+            ],
+            'Expected credential status type to include BitstringStatusList.');
+          });
+          it('The value of the purpose property of the status entry, ' +
+            'statusPurpose, MUST be one or more strings.',
+          async function() {
+            const statusPurposeType = typeof (
+              statusListCredential.credentialSubject.statusPurpose);
+            statusPurposeType.should.be.oneOf(['string', 'object']);
+            if(statusPurposeType === ('object')) {
+              const credentialSubject = statusListCredential.credentialSubject;
+              credentialSubject.statusPurpose.should.be.an(
+                'array');
+              credentialSubject.statusPurpose.forEach(
+                item => chai.assert.isString(item));
+            }
+          }
+          );
+          it('The encodedList property of the credential subject MUST ' +
+            'be a Multibase-encoded base64url (with no padding) [RFC4648] ' +
+            'representation of the GZIP-compressed [RFC1952] bitstring ' +
+            'values for the associated range of verifiable credential status ' +
+            'values.',
           async function() {
-            this.test.cell = {columnId: issuerName, rowId: this.test.title};
             const {slc: {credentialSubject}} = await getSlc({issuedVc});
             const {encodedList} = credentialSubject;
             // Uncompress encodedList
             const decoded = await sl.decodeList({encodedList});
             should.exist(decoded);
```

Comment:
```suggestion
            decoded.should.be.an('array');
```

The `.exist` property, while available, is generally frowned upon... It also doesn't support error message passing which `.a()` (or `.an()`) does: https://www.chaijs.com/api/bdd/#exist